src/transforms/transform-deployment.ts

```diff
@@ -219,7 +248,8 @@ export const transformDeployment = async (githubClient: GitHubAPI, newGitHubClie
 				id: deployment_status.environment,
 				displayName: deployment_status.environment,
 				type: environment
-			}
+			},
+			associations: associations
```

Comment:
in typescript, you don't have to duplicate the name of key and value if they're the same, you can just do `associations` and it does it for you :)